source/agora/node/Ledger.d

```diff
@@ -80,6 +80,10 @@ public class Ledger
     /// a block was externalized
     private void delegate () nothrow @safe onValidatorsChanged;
 
+    /// If not null call this delegate
+    /// A block was externalized
```

Comment:
```suggestion
    /// If not null call this delegate when a new block was added to the ledger
```

It does not necessarily mean "externalized", because this delegate will be called when a node is in catch-up mode. For example when calling `getBlocksFrom`.

This is why I don't like to call the handler "BlockExternalizedHandler".

However we don't have to worry about naming now, we can change it later.

Reply:
I also considered not focusing on Block Externalized. I think a few word have been added.